### PR TITLE
Add a control to the apps popup that allows the user to "reset apps" …

### PR DESCRIPTION
**Pull request by Perumal605**
…It will delete the user's app data so they get the global app data

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208185785540386

### CODE REVIEW

File: resources/js/components/home-footer.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ import '@spectrum-web-components/overlay/overlay-trigger.js'
 import { customElement } from 'lit-element'
 import './app-menu.js'
 import './app-menu-item.js'
-import { magic_url } from '../helpers.js'
```
Comment (incraigulous) [inline resources/js/components/home-footer.js:9]: @Perumal605 , why did y. ou remove the magic_url helper? This was added on purpose to fix a bug where the magic URL may not be applied properly

File: resources/js/components/home-footer.js [hunk 2]
```diff
@@ -234,7 +248,7 @@ class HomeFooter extends LitElement {
     }
 
     postAppDataToServer(appSlug) {
-        const url = magic_url('unhide')
```
Comment (incraigulous) [inline resources/js/components/home-footer.js:237]: @Perumal605 , please use the magic_url helper, also, notice the route has changed. It is currently "unhide", not "un-hide-app"

File: resources/js/components/home-footer.js [hunk 3]
```diff
@@ -278,8 +291,37 @@ class HomeFooter extends LitElement {
         return /^(https?:\/\/|data:image|\/|\.\/|\.\.\/)/.test(icon)
     }
 
+    reset_apps() {
+        const confirmDelete = confirm(
+            'Are you sure you want to reset all apps?'
+        )
+
+        if (confirmDelete) {
+            fetch(this.appUrl + '/reset-apps', {
```
Comment (incraigulous) [inline resources/js/components/home-footer.js:300]: @Perumal605 , Please use the magic_url() helper.

**Review by incraigulous — CHANGES_REQUESTED** [review]

File: resources/js/components/home-footer.js (hunk 1 above)
Comment (Perumal605) [reply resources/js/components/home-footer.js]: @incraigulous I have updated magic_url helper.

File: resources/js/components/home-footer.js (hunk 2 above)
Comment (Perumal605) [reply resources/js/components/home-footer.js]: @incraigulous I have updated magic_url helper.

File: resources/js/components/home-footer.js (hunk 3 above)
Comment (Perumal605) [reply resources/js/components/home-footer.js]: @incraigulous Now I am using the Magic_url helper.

**Review by incraigulous — APPROVED** [review]

**Pull request merged**